### DISCUSSION TITLE
Is it possible to make an ideaHelix plugin like ideaVim for JetBrains IDE

### DISCUSSION THREAD

**Discussion by zhaob1n** [helix-editor/helix#2432] [Ideas]
I wish I could use helix-style editor in all jetbrains products.

**Comment by ristomatti** [comment]
You can emulate Helix to some extent with IdeaVim:
- https://github.com/chtenb/helix.vim
- https://www.reddit.com/r/HelixEditor/comments/14aqztf/comment/jocslxs/
